File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/MedlemskapA004.java
package no.nav.melosys.eessi.models.sed.medlemskap.impl;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import no.nav.melosys.eessi.models.sed.medlemskap.Medlemskap;
import no.nav.melosys.eessi.models.sed.nav.Avslag;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class MedlemskapA004 implements Medlemskap {

    private Avslag avslag;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/MedlemskapA004.kt
package no.nav.melosys.eessi.models.sed.medlemskap.impl

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.melosys.eessi.models.sed.medlemskap.Medlemskap
import no.nav.melosys.eessi.models.sed.nav.Avslag

@JsonIgnoreProperties(ignoreUnknown = true)
data class MedlemskapA004(
    var avslag: Avslag? = null
) : Medlemskap
